.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
+-->
+
+* Description of issue
+
+    description
+
+* `python -c "import niscope; niscope.print_diagnostic_information()"` output (replace niscope with appropriate package name)
+
+    ```
+    replace with output
+    ```
+
+* Steps to reproduce issue
+
+    ``` python
+    Script to reproduce if applicable
```

Comment:
To clarify intent of replacing `Script to reproduce if applicable`, maybe switch it to `<Script to reproduce if applicable>`

Reply:
Changed